### PR TITLE
Advanced mode displaying #form even without flags on

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?225058

### CODE REVIEW

File: src/expressionEditor.js [hunk 1]
```diff
@@ -355,8 +355,6 @@ define([
             } else {
                 showAdvancedMode(options.value);
             }
-
-            $div.find(".fd-xpath-editor-text").val(options.value);
```
Comment (orangejenny) [inline src/expressionEditor.js]: It appears that this line is now redundant with `setExpression`.

**Comment by millerdev** [comment]
I assume you tested locally. Writing tests for the expression editor is hard, but [it is possible](https://github.com/dimagi/Vellum/blob/3c2be45002270ef3f98fa156800911135ecb09cc/tests/widgets.js#L43-L64). Understandable if you looked at that and just gave up.

**Comment by orangejenny** [comment]
I did test locally, but was admittedly lazy about writing a test for this. I'll take a look at adding one.

**Comment by orangejenny** [comment]
@emord graciously wrote a test in https://github.com/dimagi/Vellum/pull/641, so we're good for test coverage here.

**Comment by millerdev** [comment]
yay!

**Pull request merged**